share/completions/adduser.fish

```diff
@@ -15,7 +15,7 @@ complete -c adduser -l gid -d 'When creating a group, force the groupid to be th
 complete -c adduser -l group -d 'Create a group'
 complete -c adduser -l help -d 'Display brief instructions'
 complete -c adduser -l home -d 'Use specified directory as the user\'s home directory' -x -a '(__fish_complete_directories)'
-complete -c adduser -l shell -d 'Use shell as the user\'s login shell, rather than the default' -x -a '(cat /etc/shells)'
+complete -c adduser -l shell -d 'Use shell as the user\'s login shell, rather than the default' -x -a '(string match --regex --invert "^\s*#" < /etc/shells)'
```

Comment:
Nice one. The `chsh` completions use `string match -r '^[^#].*'`, which I think is slightly more correct (based on FreeBSD's documentation and the source for Debian's `shadow`).

Reply:
Thanks. ;)